crates/language_model/src/provider/anthropic.rs

```diff
@@ -21,7 +21,7 @@ use crate::{
     LanguageModelRequest, LanguageModelRequestMessage, Role,
 };
 
-const PROVIDER_NAME: &str = "anthropic";
+const PROVIDER_NAME: &str = "Anthropic";
```

Comment:
I think we'll want to leave these programmatic identifiers the way they are and replace them with user-facing strings when presenting them in the UI.

Reply:
Uhm, yeah, I was concerned this could break something. I'd appreciate tips on how to do it—I had initially thought that something akin to CSS's capitalize would work well, but then "OpenAI" 😅

Reply:
I pushed up https://github.com/zed-industries/zed/pull/15056/commits/a53bb7108f47480e700e684a943794b898c6e721 with a change.

We can match on the provider names and replace them however we wish for display purposes 🙂

Reply:
We decided to fix this a different way an introduce separate types for ids and names on the provider level